scripts/lint-dependencies.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * This script finds all the `import` and `require` of a packages
+ * and checks that there's a dependency for each one of them in
+ * the `package.json`.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const globby = require('globby');
+const builtIn = require('builtin-modules');
+
+const ignoredDependencies = new Set([
+    'har-format',
+    'estree'
+]);
+
+const getPackages = async () => {
+    const pkgs = await globby(['packages/*'], {
+        gitignore: true,
+        onlyFiles: false
+    });
+
+    console.log(`Packages found: ${pkgs.length}`);
+
+    return pkgs.map((pkg) => {
```

Comment:
if you use the option `absolute: true` in `globby` you won't need this `.map`.

Reply:
Updated!